specs/sharding/beacon-chain.md

```diff
@@ -544,6 +597,33 @@ the length proof is the commitment to the polynomial `B(X) * X**(MAX_DEGREE + 1
 where `MAX_DEGREE` is the maximum power of `s` available in the setup, which is `MAX_DEGREE = len(G2_SETUP) - 1`.
 The goal is to ensure that a proof can only be constructed if `deg(B) < l` (there are not hidden higher-order terms in the polynomial, which would thwart reconstruction).
 
+##### Shard Proposer slashings
+
+```python
+def process_shard_proposer_slashing(state: BeaconState, proposer_slashing: ShardProposerSlashing) -> None:
+    header_1 = proposer_slashing.signed_header_1.message
```

Comment:
These are "references" not "headers" in the current terminology. I suggest var names be consistent with current type names even though that will likely change

Reply:
e.g. `reference_1`, `reference_2`